telemetry/specs/INT.mdk

```diff
@@ -494,19 +494,140 @@ corruption at preceding hops.
+5. *INT over Geneve* - Geneve is an extensible tunneling framework, allowing
 Geneve options to be defined for INT Headers.
 
+### INT over IPv6
+
+In case the traffic being monitored is not encapsulated by any virtualization
+header, INT over VXLAN or INT over Geneve is not helpful. Instead, 
+the INT metadata can be carried as Hop-by-Hop option data. 
+
+IPv6 Hop-by-Hop Option format for carrying INT Header and 
+Metadata:
+
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+<-+
+|  Nxt HDR = UL | HbyH Ext Len  |           Padding|(MBZ)       |  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+  I
+| Option Type   | Opt Data Len  |        Reserved (MBZ)         |  N
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+  T
+|   Variable Option Data (INT Metadata Headers and Metadata)    |  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+<-+
+
+
+Nxt Hdr: 8-bit selector. Identifies the type of header immediately following the 
+Hop-by-Hop or Destination Options header. Uses the same values as the IPv4 Protocol
+Field [IANA-PN]
+
+HDR Ext Len: 8-bit unsigned integer. Length of the Hop-by-Hop or Destination
+Options header in 8-octet units, not including the first 8 octets.
+
+Option Type: 8-bit identifier of the type of option.
+
+  001xxxxxx 8-bit identifier of the type of option. xxxxxx=TBD_IANA_INT_HOP_BY_HOP_OPTION_IPV6.
+  001xxxxxx 8-bit identifier of the type of option. xxxxxx=TBD_IANA_INT_DESTINATION_OPTION_IPV6.
+
+Opt Data Len: 8-bit unsigned integer. Length of the Reserved and Option Data field of this 
+option, in octets.
+
+Reserved (MBZ): 16 bit field, must be filled with zeroes.
+Variable Opt Data: INT Header and Metadata, multiple of four octets in length.
+
+Padding: 16-bit pad. Needed to ensure that the variable length of the complete 
+Hop-by-Hop or Destination Options Header is an integer multiple of 8 octets long.
+
+
+The format of the IPv6 packet with Hop-by-Hop option is shown below:
+
+0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|Version| Traffic Class |            Flow Label                 |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|     Payload Length            |Nxt HDR = HbyH |   Hop Limit   |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                       Source IPv6 Address                     |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                    Destination IPv6 Address                   |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+<-+
+|  Nxt HDR = UL | HbyH Data Len |            Padding            |  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+  I	
+|  Option Type  |  Opt Data Len |          Reserved (MBZ)       |  N
```

Comment:
Throw in INT/IOAM Type field, as noted in comment above. Same for all the following packet examples.

Reply:
Done.